Python/specialize.c

```diff
@@ -831,9 +833,7 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name)
             assert(type->tp_getattro == _Py_slot_tp_getattro);
             assert(Py_IS_TYPE(descr, &PyFunction_Type));
             _PyLoadMethodCache *lm_cache = (_PyLoadMethodCache *)(instr + 1);
-            uint32_t func_version = function_check_args(descr, 2, LOAD_ATTR) &&
-                function_get_version(descr, LOAD_ATTR);
-            if (func_version == 0) {
+            if (!function_check_args(descr, 2, LOAD_ATTR)) {
```

Comment:
Should we write the function version into the 16 bit `dk_version` field like we were discussing the other day?

Reply:
Maybe, but probably in another PR that checks version for all specialized calls.